app/views/landing_page/blocks/_main_navigation.html.erb

```diff
@@ -1,45 +1,46 @@
 <%
   add_view_stylesheet("landing_page/main-navigation")
 %>
+<div class="app-b-main-nav--border">
+  <div class="app-b-main-nav govuk-width-container" data-module="app-b-main-navigation">
```

Comment:
Seems like the BEM's got a bit confusing here... `app-b-main-nav--border` is a modifier class, but it's being used as the parent class. I think the first class that wraps the whole block should be `app-b-main-nav` and then the inner div (with the govuk width container) should be an element class, like `app-b-main-nav__wrapper` or something. Will need to juggle the actual styles, obviously.

Reply:
Thanks, done 👍